### PR TITLE
Safe codegen formatting changes 

### PR DESCRIPTION
**Pull request by rattrayalex-stripe**
Supersedes https://github.com/stripe/stripe-python/pull/565

Just a rebase & squash

r? @ob-stripe 
cc @dmitrykorolev-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: stripe/api_resources/account.py [hunk 1]
```diff
@@ -3,11 +3,11 @@
 import stripe
 from stripe import oauth, six, util
 from stripe.api_resources.abstract import CreateableAPIResource
+from stripe.api_resources.abstract import custom_method
```
Comment (ob-stripe) [inline stripe/api_resources/account.py]: Nit: I think the sort order should be case-sensitive, i.e. `custom_method` should appear after `UpdateableAPIResource`.

No need to act specifically for this though, as I expect `custom_method` will disappear fairly soon and we'll just generate the methods directly instead.

**Comment by rattrayalex-stripe** [comment]
Thanks ob! I think you're right about the sorting, gonna try to take a quick crack at that. 

Looks like the deepsource warnings are quite ignorable, I assume that's ok.

**Comment by ob-stripe** [comment]
> Looks like the deepsource warnings are quite ignorable, I assume that's ok.

Yes, feel free to ignore -- I enabled it to play around with it and see what sort of issues it would report, but I didn't realize it would add a check on PRs.

**Comment by rattrayalex-stripe** [comment]
Gonna hold off on this for a bit since I'm quite close to having nested methods working in Python

**Comment by rattrayalex-stripe** [comment]
Okay, added two more commits which are small & independently reviewable. 

r? @ob-stripe

**Comment by rattrayalex-stripe** [comment]
Done! Thanks ob!

**Pull request merged**